chapters/ch14.md

```diff
@@ -1,54 +1,75 @@
+
 
 In detail, here’s what you can do:
 ->
-
-
 1. Gather a sample of 100 dev set examples that your system ​misclassified​. I.e., examples that your system made an error on.
 ->
-
-
 2. Look at these examples manually, and count what fraction of them are dog images.
 ->
 
+Cụ thể, bạn có thể làm theo các bước sau:
+
+1. Thu thập  gồm 100 mẫu trong tập phát triển mà ứng dụng của bạn phân loại nhầm — không phải mèo nhưng được gán nhãn mèo và ngược lại
+
+2. Nhìn vào những mẫu trên và đếm xem bao nhiêu trong số đó là ảnh chó
 
 The process of looking at misclassified examples is called ​**error analysis​**. In this example, if you find that only 5% of the misclassified images are dogs, then no matter how much you improve your algorithm’s performance on dog images, you won’t get rid of more than 5% of your errors. In other words, 5% is a “ceiling” (meaning maximum possible amount) for how much the proposed project could help. Thus, if your overall system is currently 90% accurate (10% error), this improvement is likely to result in at best 90.5% accuracy (or 9.5% error, which is 5% less error than the original 10% error).
 ->
 
+Quá trình nhìn vào những mẫu bị gán nhãn nhầm được gọi là phân tích lỗi (2). Trong ví dụ trên, nếu bạn nhấn thấy rằng chỉ 5% trong số những bức ảnh bị nhầm từ chó thành mèo thì cho dù cải thiện thuật toán theo hướng tích hợp phần mềm nhận dạng chó vào ứng dụng, bạn không thể loại bỏ quá 5% số ảnh bị nhận dạng sai. Nói cách khác, 5% là “cận trên” (số lượng tối đa có thể đạt được) cho mức độ cải thiện mà hướng đi trên có thể giúp cho hệ thống. Nếu như độ chính xác ban đầu của ứng dụng là 90% (10% lỗi), việc cải thiện chỉ làm cho hệ thống của bạn đạt được độ chính xác mới là 90.5% (9.5% lỗi, ít hơn 5% so với số lượng lỗi ban đầu). 
+
 
 In contrast, if you find that 50% of the mistakes are dogs, then you can be more confident that the proposed project will have a big impact. It could boost accuracy from 90% to 95% (a 50% relative reduction in error, from 10% down to 5%).
 ->
 
+Ngược lại, nếu bạn nhận thấy rằng 50%  lỗi là do chó bị nhầm thành mèo thì bạn có thể tự tin rằng phương án được đề xuất sẽ cải thiện đáng kể độ chính xác của hệ thống từ 90% lên 95% (giảm 50% tổng số lỗi, từ 10% xuống 5%) 
 
 This simple counting procedure of error analysis gives you a quick way to estimate the possible value of incorporating the 3rd party software for dog images. It provides a quantitative basis on which to decide whether to make this investment.
 ->
 
+Cách phân tích lỗi đơn giản ở trên giúp bạn ước lượng nhanh hiệu quả của việc tích hợp phần mềm nhận dạng chó của một công ty khác vào hệ thống nhận dạng mèo của nhóm. Nó cũng là cơ sở định lượng để bạn lựa chọn xem có nên đi theo hướng này hay không.
```

Comment:
* Mình thấy bạn hay dịch "this" thành "ở trên" hay "trên đây". Mình thấy dịch là "này" không có gì khó hiểu.
* "vào hệ thống nhận dạng mèo ~~của nhóm~~."

Reply:
ok